Week 9/Marvel/app/src/main/java/com/carol/musyoka/marvel/marvelapi/splash/ITimerSplash.kt

```diff
@@ -0,0 +1,7 @@
+package com.carol.musyoka.marvel.marvelapi.splash
+
```

Comment:
Your splashscreen works great and looks really cool, I have seen another approach to splas screens like this one just for reference https://android.jlelse.eu/right-way-to-create-splash-screen-on-android-e7f1709ba154#:~:text=As%20we%20all%20know%20splash,pattern%20known%20called%20Launch%20Screen.

Reply:
Thank you. I will check it out .